src/lib/CMakeLists.txt

```diff
@@ -46,6 +46,19 @@ set_target_properties(partio PROPERTIES
     POSITION_INDEPENDENT_CODE ON
     SOVERSION ${PARTIO_VERSION_MAJOR})
 
+set(TBB_INCLUDE_DIRS "$ENV{TBB_ROOT}/include")
+set(CMAKE_MODULE_PATH "$ENV{OpenVDB_ROOT}/lib/cmake/OpenVDB"
+    "${OpenVDB_ROOT}/lib/cmake/OpenVDB"
+    ${CMAKE_MODULE_PATH})
+find_package(OpenVDB COMPONENTS openvdb)
+find_package(Alembic)
+find_package(Boost)
+find_package(TBB)
+
+
+target_link_libraries(partio PRIVATE Alembic::Alembic OpenVDB::openvdb)
+target_include_directories(partio PRIVATE ${TBB_INCLUDE_DIRS})
+
```

Comment:
Right now partio has a really minimal set of dependencies.

Since these are new dependencies it would be good to ease into them. Can we please make these opt-in and not built by default?

I would suggest adding some `option(...)` variables that default to `OFF`. Maybe call them something like `PARTIO_ENABLE_ALEMBIC` and `PARTIO_ENABLE_OPENVDB` so that we can opt-in to these settings.

TBB is needed for openvb so we can bundle those settings alongside `PARTIO_ENABLE_OPENVDB`.